testing/running-tests.adoc

```diff
@@ -61,15 +61,16 @@ way.
 === Running A Single File
 
 If you want to run a single test file, say, `t/foo.t`, then all you need
-to do is just typing the following command in your terminal.
+to do is just type the following command in your terminal.
```

Comment:
I don't agree with this one. It seems wrong to use the original form of verbs directly after `is`. @ingydotnet What do you think?

Maybe we should use `is just to type ...` here instead?